src/widget/transfer_function.ts

```diff
@@ -1602,9 +1602,7 @@ class TransferFunctionController extends RefCounted {
  * Widget for the transfer function. Creates the UI elements required for the transfer function.
  */
 class TransferFunctionWidget extends Tab {
-  private transferFunctionPanel = this.registerDisposer(
-    new TransferFunctionPanel(this),
-  );
+  private transferFunctionPanel = this;
```

Comment:
Should not be assigned to `this`

Reply:
Thanks for spotting this @jbms :pray: it's a bad copy/paste...